charts/nexus-repository-manager/templates/deployment.yaml

```diff
@@ -57,8 +57,6 @@ spec:
       {{- end }}
       containers:
         - name: {{ .Chart.Name }}
-          securityContext:
-            {{- toYaml .Values.securityContext | nindent 12 }}
```

Comment:
lose the single container's securityContext, since it's set and documented at the pod level.